robottelo/cli/import_.py

```diff
@@ -0,0 +1,157 @@
+# -*- encoding: utf-8 -*-
+"""
+Usage::
+
+    hammer import [OPTIONS] SUBCOMMAND [ARG] ...
+
+Parameters::
+
+    SUBCOMMAND                    subcommand
+    [ARG] ...                     subcommand arguments
+
+Subcommands::
+
+    activation-key                Import Activation Keys
+                                  (from spacewalk-report activation-keys).
+    all                           Load ALL data from a specified directory
+                                  that is in spacewalk-export format.
+    config-file                   Create puppet-modules from Configuration
+                                  Channel content (from spacewalk-report
+                                  config-files-latest).
+    content-host                  Import Content Hosts
+                                  (from spacewalk-report system-profiles).
+    content-view                  Create Content Views based on local/cloned
+                                  Channels (from spacewalk-export-channels).
+    host-collection               Import Host Collections
+                                  (from spacewalk-report system-groups).
+    organization                  Import Organizations
+                                  (from spacewalk-report users).
+    repository                    Import repositories
+                                  (from spacewalk-report repositories).
+    repository-enable             Enable any Red Hat repositories
+                                  accessible to any Organization
+                                  (from spacewalk-report channels).
+    template-snippet              Import template snippets
+                                  (from spacewalk-report kickstart-scripts).
+    user                          Import Users (from spacewalk-report users).
+
+"""
+from robottelo.cli.base import Base
+
+
+class Import(Base):
+    """Imports configurations from another Satellite instances."""
+    command_base = 'import'
+
+    @classmethod
+    def activation_key(cls, options=None):
+        """Import Activation Keys (from spacewalk-report activation-keys).
+        Requires organization.
+
+        """
+        cls.command_sub = 'activation-key'
+
```

Comment:
nitpick: drop the whitespace between `cls.command_sub` and `return cls.execute`. Whitespace is very useful when it helps to tell the reader that there are large, significantly different logical thoughts being expressed. That's not the case here.